dotnet/csharp.bzl

```diff
@@ -407,6 +417,104 @@ def csharp_configure():
       actual = "@local_config_csharp//:csc_bin",
   )
 
+def _nuget_dep_impl(ctx):
+  inputs = set(ctx.files.srcs)
+  return struct(
+    name = ctx.label.name,
+    target_type = ctx.attr._target_type,
+    out = inputs,
+    dlls = inputs,
+    transitive_dlls = set([]),
+  )
+
+nuget_dep = rule(
```

Comment:
This seems similar to the [java_import](http://bazel.io/docs/be/java.html#java_import) rule. Do you think this rule might be useful for use cases other than nuget packages?